frontend/tests/integration/components/team-form-test.js

```diff
@@ -0,0 +1,93 @@
+import { module, test } from "qunit";
+import { setupRenderingTest } from "ember-qunit";
+import { render } from "@ember/test-helpers";
+import { hbs } from "ember-cli-htmlbars";
+import Service from "@ember/service";
+
+const storeStub = Service.extend({
```

Comment:
Maybe it makes sense to create a storeStub for all the integration tests

Reply:
This is very strange. I removed storeStub from this test, as it's not needed, but here, it's still present.